app/src/main/java/ru/netology/fmhandroid/ui/AppActivity.kt

```diff
@@ -0,0 +1,25 @@
+package ru.netology.fmhandroid.ui
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import ru.netology.fmhandroid.R
+
+class AppActivity : AppCompatActivity(R.layout.activity_app) {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        intent?.let {
```

Comment:
```suggestion
        val intent = intent ?: return
```
И у тебя сразу уйдёт лишняя вложенность от `let`, код сместится влево и будет занимать меньше места

P.S. У `if`-ов ниже можно тоже убрать блочные скобки: `if (...) return`

Reply:
Отлично )
Исправлено.

Reply:
Исправлено